docs/user/whats-new.asciidoc

```diff
@@ -11,4 +12,91 @@ coming[8.0.0]
 
 // tag::notable-highlights[]
 
+[float]
+=== Security on-by-default
```

Comment:
```suggestion
=== Security features are enabled and configured by default
```

Reply:
We refer to this feature as "security on-by-default" internally, but I think that externally we want to indicate that security features are enabled and configured by default. That's the approach we're taking in the release highlight for the Elasticsearch release notes: https://github.com/elastic/elasticsearch/pull/83292/files#diff-6084df60d0e64f7e1c8b55063a278fb627eda6ace267952dba756f6bf4912bf7R19